src/core/lwt.ml

```diff
@@ -893,35 +848,33 @@ struct
 
     (* Go through the promises the cell had originally been added to, and either
        defer a cleanup, or actually clean up their callback lists. *)
-    ps |> List.iter (fun p ->
-      let Internal p = to_internal_promise p in
-      match (underlying p).state with
-      (* Some of the promises may already have been resolved at the time this
-         function is called. *)
-      | Fulfilled _ -> ()
-      | Rejected _ -> ()
-
-      | Pending callbacks ->
-        match callbacks.regular_callbacks with
-        (* If the promise has only one regular callback, and it is removable, it
-           must have been the cell cleared in this function, above. In that
-           case, just set its callback list to empty. *)
-        | Regular_callback_list_explicitly_removable_callback _ ->
-          callbacks.regular_callbacks <- Regular_callback_list_empty
-
-        (* Maintainer's note: I think this function shouldn't try to trigger a
-           cleanup in the first two cases, but I am preserving them for now, as
-           this is how the code was written in the past. *)
-        | Regular_callback_list_empty
-        | Regular_callback_list_implicitly_removed_callback _
-        | Regular_callback_list_concat _ ->
-          let cleanups_deferred = callbacks.cleanups_deferred + 1 in
-          if cleanups_deferred > cleanup_throttle then begin
-            callbacks.cleanups_deferred <- 0;
-            callbacks.regular_callbacks <-
-              clean_up_callback_cells callbacks.regular_callbacks
-          end else
-            callbacks.cleanups_deferred <- cleanups_deferred)
+    ps
+    |> List.iter (fun p ->
+           let (Internal p) = to_internal_promise p in
```

Comment:
This is not a helpful comment, but I'm sad about this. Unlike most code, `lwt.ml` is carefully formatted to look good, and ocamlformat is introducing things such as this indentation.

Reply:
To make it potentially somewhat helpful, in the past [we applied ocp-indent](https://github.com/ocsigen/lwt/pull/409) (then the go-to formatting tool) to the almost the whole repo, but we avoided touching code that had gotten the full human attention.